federation/cmd/federated-apiserver/app/server.go

```diff
@@ -36,6 +35,9 @@ import (
 	"k8s.io/kubernetes/pkg/admission"
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/unversioned"
+	"k8s.io/kubernetes/pkg/apis/autoscaling"
```

Comment:
These were removed in https://github.com/kubernetes/kubernetes/pull/23959.
We shouldnt be adding them back :)